xgcm/test/test_grid.py

```diff
@@ -3,20 +3,107 @@
 import pytest
 import xarray as xr
 import numpy as np
+import dask as dsk
 from dask.array import from_array
 
 from xgcm.grid import Grid, Axis, add_to_slice
+from xgcm.duck_array_ops import concatenate
 
 from . datasets import (all_datasets, nonperiodic_1d, periodic_1d, periodic_2d,
                         nonperiodic_2d, all_2d, datasets)
 
+
 # helper function to produce axes from datasets
 def _get_axes(ds):
     all_axes = {ds[c].attrs['axis'] for c in ds.dims if 'axis' in ds[c].attrs}
     axis_objs = {ax: Axis(ds, ax) for ax in all_axes}
     return axis_objs
 
 
+# duck array ops should maybe get its own test module?
```

Comment:
👍

Reply:
I would be happy if you would factor those out into a new test module.

Reply:
Done